site/cypress/e2e/profile.spec.js

```diff
@@ -71,6 +71,61 @@ describe('Test cases revolving around user profile page', () => {
         cy.get('#secondary-email-notify-row').should('be.visible');
     });
 
+    // Selenium test_basic_info
+    it('Should start with accurate values', () => {
+        cy.get('[data-testid="username-row"]').should('contain.text', 'instructor');
+        cy.get('[data-testid="givenname-row"]').should('contain.text', 'Quinn');
+        cy.get('[data-testid="familyname-row"]').should('contain.text', 'Instructor');
+        cy.get('[data-testid="pronouns-row"]').should('contain.text', 'She/Her');
+        cy.get('[data-testid="email-row"]').should('contain.text', 'instructor@example.com');
+    });
+
+    // Selenium test_time_zone_selection
+    it('Time zone selector should work', () => {
+        // 420 time zones
+        const random_number = Math.floor(Math.random() * 420);
```

Comment:
Random numbers in tests are *BAD*.  What if this test fails in one single case out of the possible 420?  Rerunning the test would likely make it pass and look like a flaky test.  Instead of this, check fixed edge cases which can be deterministically checked.

Reply:
Wouldn't it be good to catch that single case rather than testing just a few cases? The old selenium tests chose one of the time zones at random,   `# Now check if updating the time_zone is correctly working
        # Choose a random option and select it`

Reply:
I changed this away from random selection.

Reply:
> Wouldn't it be good to catch that single case rather than testing just a few cases? The old selenium tests chose one of the time zones at random, `# Now check if updating the time_zone is correctly working # Choose a random option and select it`

Yes, if there are edge cases, you definitely want to have a specific test case for them.  You just don't want to risk running into an unforeseen edge case accidentally by making the test non-deterministic.